smf/nexus/multi-sled/config-partial.toml

```diff
@@ -76,6 +76,9 @@ region_snapshot_replacement_step.period_secs = 30
 region_snapshot_replacement_finish.period_secs = 30
 tuf_artifact_replication.period_secs = 300
 tuf_artifact_replication.min_sled_replication = 3
+tuf_repo_pruner.period_secs = 300
+tuf_repo_pruner.nkeep_extra_target_releases = 1
+tuf_repo_pruner.nkeep_extra_newly_uploaded = 1
```

Comment:
Is this the config we ultimately want to have? (I remember discussing setting these each to 3 or the like but I could be misremembering.)

Reply:
Oh these are in _addition_ to. Yeah, a comment would be good. (While reading through I had somehow thought the maximum of the hardcoded minimum and the configured amount would be used.)

Reply:
Added a comment.